src/assets/main.scss

```diff
@@ -731,4 +731,14 @@ input {
     .inactive .account.card {
         opacity: 0.6;
     }
+}
+
+.c-toast {
+  position: relative;
+
+  .delete {
+    position: absolute;
+    top: .4rem;
+    right: .4rem;
+  }
```

Comment:
is this still necessary if you don't have any toaster introduced in this commit ? Same for the commit message, that seems to need to be updated.

Reply:
Thanks for the heads-up.

Changed.